### PR TITLE
Added SLF4j, logback  dependencies in the pom.xml file and implemente…

### PR DESCRIPTION
**Pull request by ArevikArakelyan1**
…d methods

### CODE REVIEW

File: src/main/java/business/SharedData.java [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public final class SharedData {
+    private final static Logger logger = LoggerFactory.getLogger(SharedData.class);
     private static String token;
-
     public static String getToken() {
+        logger.info("GetToken is {}",token);
```
Comment (mkrtichstepanyan) [inline src/main/java/business/SharedData.java:10]: remove

File: src/main/java/business/SharedData.java [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
         return token;
     }
-
     public static void setToken(String token) {
         SharedData.token = token;
+        logger.info("SetToken is {}",SharedData.token);
```
Comment (mkrtichstepanyan) [inline src/main/java/business/SharedData.java:15]: remove

File: src/main/java/business/RequestsUtils.java [hunk 3]
```diff
@@ -5,29 +5,34 @@
 import io.restassured.http.ContentType;
 import io.restassured.response.ValidatableResponse;
 import io.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class RequestsUtils {
```
Comment (mkrtichstepanyan) [inline src/main/java/business/RequestsUtils.java:11]: reformate code

File: src/main/java/business/RequestsUtils.java [hunk 4]
```diff
@@ -5,29 +5,34 @@
...
                     .given()
                     .when()
                     .get(endpoint)
                     .then();
-    }
+         logger.info(response.extract().asPrettyString());
 
+    }
     public static void post(String endpoint, Object body){
+        logger.info("Before posting User");
```
Comment (mkrtichstepanyan) [inline src/main/java/business/RequestsUtils.java:27]: remove

**Review by mkrtichstepanyan — CHANGES_REQUESTED** [review]

File: src/main/resources/logback.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        <encoder>
+            <pattern>
+                %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
+            </pattern>
+        </encoder>
+    </appender>
+    <logger name="MyLogger" level="info" >
+        <appender ref="MyConsoleAppender"/>
+        <appender ref="MyFileAppender"/>
+    </logger>
```
[the remark this answers is not in the document]
Comment (ArevikArakelyan1) [reply src/main/resources/logback.xml:22]: Done

File: src/test/java/steps/CommonSteps.java [hunk 6]
```diff
@@ -5,24 +5,30 @@
...
 import io.restassured.RestAssured;
 import org.assertj.core.api.Assertions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class CommonSteps {
-
+    private final Logger logger = LoggerFactory.getLogger(CommonSteps.class);
     private static final String BASE_URL = "http://open-school-dev.eu-central-1.elasticbeanstalk.com/";
     private static final String BASE_PATH = "api/v1/";
```
[the remark this answers is not in the document]
Comment (ArevikArakelyan1) [reply src/test/java/steps/CommonSteps.java:14]: Done

File: pom.xml [hunk 7]
```diff
@@ -81,5 +82,52 @@
             <artifactId>json-path</artifactId>
             <version>5.1.1</version>
         </dependency>
+        <!-- allure-Report -->
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-cucumber-jvm</artifactId>
+            <version>2.19.0</version>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (ArevikArakelyan1) [reply pom.xml:90]: Done

File: src/test/java/steps/CommonSteps.java [hunk 8]
```diff
@@ -5,24 +5,30 @@
...
-
     @Given("Setup Rest Assured")
     public void setupRestAssured() {
         RestAssured.baseURI = BASE_URL;
         RestAssured.basePath = BASE_PATH;
+        logger.info("The BASE_URL is {}" ,BASE_URL);
+        logger.info("The BASE_PATH is {}",BASE_PATH);
```
[the remark this answers is not in the document]
Comment (ArevikArakelyan1) [reply src/test/java/steps/CommonSteps.java:20]: done

File: src/main/java/business/SharedData.java [hunk 9]
```diff
@@ -1,13 +1,17 @@
 package business;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public final class SharedData {
```
[the remark this answers is not in the document]
Comment (ArevikArakelyan1) [reply src/main/java/business/SharedData.java:6]: done

File: src/main/java/business/SharedData.java (hunk 2 above)
Comment (ArevikArakelyan1) [reply src/main/java/business/SharedData.java:15]: done

**Review by mkrtichstepanyan — APPROVED** [review]

**Pull request merged**